package.json

```diff
@@ -51,6 +51,7 @@
     "packages/angular/projects/ui-angular"
   ],
   "resolutions": {
+    "aws-amplify": "latest",
```

Comment:
I would prefer update this under each package itself. IMO, `resolutions` is the last resort to pin transitive dependencies